src/wp-includes/load.php

```diff
@@ -239,6 +239,49 @@ function wp_get_environment_type() {
 	return $current_env;
 }
 
+/**
+ * Retrieves the current runtime environment.
+ *
+ * The value can be set via the `WP_RUNTIME_ENVIRONMENT` global system variable,
+ * or a constant of the same name.
+ *
+ * The only value currently supported is 'virtualbox'. If not set, the value
+ * defaults to an empty string.
```

Comment:
I'm considered about introducing a constant that must be set for enabling VirtualBox to work with this rollback feature. Why? All environments that use VirtualBox must make changes to turn on this constant.

- Does this mean that any site (local or prod) that uses VirtualBox will not have the rollback feature? And if yes, will they know that it's not available for usage? How will it bail out gracefully to protect sites and users?
- Are there any web hosts in the world who are using VirtualBox for their virtualization? If yes, how are they and/or their customers impacted by this change?
- What is the impact to tools and/or projects that use VirtualBox?

Reply:
Here's the issue opened against Chassis to add that constant https://github.com/Chassis/Chassis/issues/963